### ISSUE TITLE
'image' intentionally ignored? 

### ISSUE THREAD

**Issue by abhinavc** [vincedevendra/firebase_cloud_messenger#24]
So, I'm trying to send an image URL with my notifications.
This is supported by FCM for both iOS and Android in **either** of the following ways: 

```
{
     notification: { :title, :body, :image }, 
     android: { notification: { ...., :image, .... } }, 
     apns: { :headers, :payload, fcm_options: { :image } } 
}
 
```
But it seems `:image` is consistently ignored. The accessible attributes in the gem's code
are frozen to include everything other than `:image`

Can this be changed? Or, has this been done intentionally for some reason? 

Thanks for your help
Abhinav.

**Comment by vincedevendra** [comment]
I can absolutely be changed! `image` was not part of the API when the code was written. I suspect there are other new bits or changes in the API that should be reflected in the code. You're welcome to put in a PR, or I can take a look at it the next couple of days.

**Comment by abhinavc** [comment]
Hi Vince,
Thought I would use this to issue my first PR ever. But I think @ezilocchi beat me to it (#25). So, rather than duplicate effort, I'll just await the next gem release. 

Thanks for your help,
Abhinav.

**Comment by ezilocchi** [comment]
Hey @abhinavc I'm sorry I was not trying to steal your idea :P, but I'm also blocked due to this issue, that's why I move forward with this implementation.
With that said If you still want to submit your first PR on this repo there are some new fields that I haven't added on my PR and that are missing on the gem. For instance on the [Android Notification Section](https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#AndroidNotification):
```
{
    "message": {
        "android": {
            "notification": {
                "ticker": string,
                "sticky": boolean,
                "event_time": string,
                "local_only": boolean,
                "notification_priority": enum (NotificationPriority),
                "default_sound": boolean,
                "default_vibrate_timings": boolean,
                "default_light_settings": boolean,
                "vibrate_timings": [
                    string
                ],
                "visibility": enum (Visibility),
                "notification_count": number,
                "light_settings": {
                    object (LightSettings)
                },
                "image": string // I've added this already on my PR
            }
        }
    }
}
```

**Comment by vincedevendra** [comment]
@abhinavc @ezilocchi I'd love 0.5.0 to be completely up to date with the current FCM API schema. I'm inclined to go ahead and merge #25 and then wait to release the next version until that's done. 

@abhinavc if you'd like to take make that happen, I think it'd be amazing! @ezilocchi if you're blocked, would you mind pointing to master or your fork until I get a release out in a few days?
You also should be able to send a hash literal instead of building up a request with the data objects, as long as you don't try to validate it. Less than ideal, I know, but maybe an ok short term solution.

**Comment by vincedevendra** [comment]
Resolved by #25

**Comment by ezilocchi** [comment]
I would need this feature on a release, I can't just point to `master` on this repo. Can we get this other PR in as well? https://github.com/vincedevendra/firebase_cloud_messenger/pull/27
